scripts/systemd/falco-bpf.service

```diff
@@ -8,6 +8,7 @@ Wants=falcoctl-artifact-follow.service
 Type=simple
 User=root
 Environment=FALCO_BPF_PROBE=
+ExecStartPre=/usr/bin/falco-driver-loader bpf
```

Comment:
We chose to enable download too for ebpf probe, because it has a clang dep that might not be satisfied and we don't want to enforce it.
Moreover, given that ebpf probe is not managed by the system (dkms) it's ok to manage it ourselves.

Finally, note that if the correct ebpf probe is already in place, the `falco-driver-loader` script won't do anything: https://github.com/falcosecurity/falco/blob/master/scripts/falco-driver-loader#L585